node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -4199,8 +4427,8 @@ mod tests {
         )
     }
 
-    fn make_single_node_gossip(n: u16, mode: Mode) -> (Gossip_0v1, NodeRecord) {
-        let mut debut_node = make_node_record(n, true);
+    fn make_single_node_gossip<'a>(n: u16, mode: Mode) -> (Gossip_0v1, NodeRecord) {
+        let mut debut_node: NodeRecord = make_node_record(n, true);
```

Comment:
Probably being unnecessarily type-explicit. My own rule is that if something works just fine in 90% use cases just fine and then you do it a bit differently in some other place it drags automatically attention which it however should deserve. And here it's not grounded, I think.